LayoutTests/http/tests/site-isolation/frame-access-after-window-open.html

```diff
@@ -0,0 +1,23 @@
+<!-- webkit-test-runner [ SiteIsolationEnabled=true ] -->
+<script>
+if (window.testRunner) {
+    testRunner.dumpAsText();
+    testRunner.waitUntilDone();
+}
+function runTest() {
+    openedWindow = window.open("http://localhost:8000/site-isolation/resources/opened-main.html");
+}
+function runTestIfInTestRunner() { if (window.testRunner) { runTest() } }
+
+function firstSameOriginIframeOpened() {
+    alert("firstSameOriginIframeOpened called by opened iframe, test accessing custom property: " + openedWindow[0].customProperty);
+    windowOpenedInFrame = window.open("http://127.0.0.1:8000/site-isolation/resources/opened-iframe-2.html", "openedFrameName")
```

Comment:
The test should be using `shouldBe()`, not alert() like this. As it stands, it just prints out values and nobody would know if the output means the test is passing or failing.